integration/src/test/resources/templates/kotlin-multiplatform/src/commonMain/kotlin/CommonBenchmark.kt

```diff
@@ -8,8 +8,31 @@ import kotlin.math.*
 @OutputTimeUnit(BenchmarkTimeUnit.MILLISECONDS)
 @BenchmarkMode(Mode.Throughput)
 open class CommonBenchmark {
+
+    var data = 0
+
+    val valProperty: String = "test"
+
+    final var finalProperty = "This property is final"
+
+    private var privateProperty = "This property is private"
+
+    internal var internalProperty = "This property is internal"
+
     @Benchmark
     open fun mathBenchmark(): Double {
         return log(sqrt(3.0) * cos(3.0), 2.0)
     }
-}
+
+    fun setUpMethod() {
+        // println("Setup!")
+    }
+
+    fun teardownMethod() {
+        // println("Teardown!")
+    }
+
+    private fun privateMethod() {
```

Comment:
Could you please add `internal`, `protected`, `final` methods as well, and test that all platforms behave the same when the method is annotated with @Setup/@Teardown/@Benchmark?

Reply:
Will do. How do you recommend I approach verifying similar behavior.